### PR TITLE
Sample migration 5

### PR DESCRIPTION
**Pull request by kul3r4**

### CODE REVIEW

File: src/20_Components/amp-call-tracking.html [hunk 1]
```diff
@@ -40,11 +40,9 @@
 
     We also need to set a default phone number that will be shown before the endpoint responds. This is done via an `<a>` element inside the `amp-call-tracking` tag.
   -->
-  <p>
-    <strong>Call us!</strong><br>
+    <div>Call us!</div>
```
Comment (sebastianbenz) [inline src/20_Components/amp-call-tracking.html]: Wondering: just we remove this as well? Or does it help to explain the context?

File: src/20_Components/amp-date-picker.html [hunk 2]
```diff
@@ -246,6 +237,7 @@
         input-selector="#static-picker-input">
       </amp-date-picker>
 
+  <!-- ## highlighted and blocked attributes -->
```
Comment (sebastianbenz) [inline src/20_Components/amp-date-picker.html]: Uppercase highlighted

File: src/20_Components/amp-call-tracking.html (hunk 1 above)
Comment (kul3r4) [reply src/20_Components/amp-call-tracking.html]: removed

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Comment by kul3r4** [comment]
@sebastianbenz thanks for the feedback, addressed all from above

File: src/20_Components/amp-date-picker.html [hunk 3]
```diff
@@ -264,7 +259,7 @@
       blocked="FREQ=WEEKLY;WKST=SU;BYDAY=SA,SU">
     </amp-date-picker>
 
-  <!-- ## allow-blocked-dates attribute -->
+  <!-- ## Allow-blocked-dates attribute -->
```
Comment (sebastianbenz) [inline src/20_Components/amp-date-picker.html]: Not sure about this as it's a pre-defined attribute name.

**Pull request merged**